Detectors/TRD/simulation/src/Detector.cxx

```diff
@@ -32,7 +32,13 @@ Detector::Detector(Bool_t active)
     mWion = 27.21; // Ionization energy ArCO2 (82/18)
   } else {
     LOG(FATAL) << "Wrong gas mixture";
-    // add hard exit here!
+  }
+  // Switch on TR simulation as default
+  mTRon = true;
```

Comment:
I will merge now as is, but we should make this we should make this a configuration option.

Reply:
I agree. Thanks.